.idea/codeStyles/Project.xml

```diff
@@ -1,15 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="RIGHT_MARGIN" value="100" />
```

Comment:
why remove code style config?

Reply:
Android Studio does it.

Reply:
I reconfigured this margin according to https://www.jetbrains.com/help/idea/settings-code-style.html#dfd02dd4.

Reply:
I said remove for all content on this file, this codeStyle xml is match ktlint and codeStyle gradle job, if you think change is needed, I think those tow job rule you have to re-match again.

Reply:
I run okchek and everything is fun.

Reply:
I run okchek and everything is fun.